### PR TITLE
replace all response group from enum to string

### PR DESCRIPTION
**Pull request by anivirtoway**

### CODE REVIEW

File: Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Repositories/OrderRepository.cs [hunk 1]
```diff
@@ -25,19 +26,21 @@ public OrderRepository(OrderDbContext dbContext, IUnitOfWork unitOfWork = null)
...
         {
+            var customerOrderResponseGroup = EnumUtility.SafeParseFlags(responseGroup, CustomerOrderResponseGroup.Full);
+
             var result = await CustomerOrders.Where(x => ids.Contains(x.Id)).ToArrayAsync();
             var orderDiscounts = Discounts.Where(x => ids.Contains(x.CustomerOrderId)).ToArrayAsync();
             var orderTaxDetails = TaxDetails.Where(x => ids.Contains(x.CustomerOrderId)).ToArrayAsync();
             await Task.WhenAll(orderDiscounts, orderTaxDetails);
 
-            if ((responseGroup & CustomerOrderResponseGroup.WithAddresses) == CustomerOrderResponseGroup.WithAddresses)
+            if ((customerOrderResponseGroup & CustomerOrderResponseGroup.WithAddresses) == CustomerOrderResponseGroup.WithAddresses)
```
Comment (tatarincev) [inline Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Repositories/OrderRepository.cs]: Replace please to `HasFlags` expression everywhere

**Review by tatarincev — COMMENTED** [review]
And please move all ResponseGroup enum types into Data projects from Core for all modules.

File: Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Repositories/OrderRepository.cs (hunk 1 above)
Comment (artem-dudarev) [reply Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Repositories/OrderRepository.cs]: > And please move all ResponseGroup enum types into Data projects from Core for all modules.

No way!
It should be possible to call `ICustomerOrderService.GetByIdAsync("123", CustomerOrderResponseGroup.WithItems.ToString())` without adding Data project to dependencies.

**Comment by artem-dudarev** [comment]
> And please move all ResponseGroup enum types into Data projects from Core for all modules.

No way!
It should be possible to call `ICustomerOrderService.GetByIdAsync("123", CustomerOrderResponseGroup.WithItems.ToString())` without adding Data project to dependencies.

**Comment by artem-dudarev** [comment]
BTW, enums with `[Flags]` attribute should have plural names: `CustomerOrderResponseGroups`, not `CustomerOrderResponseGroup`
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1714-flags-enums-should-have-plural-names?view=vs-2019

File: Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Repositories/OrderRepository.cs (hunk 1 above)
Comment (anivirtoway) [reply Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Repositories/OrderRepository.cs]: fixed

**Review by tatarincev — APPROVED** [review]

**Pull request merged**